spec/controllers/users_controller_spec.rb

```diff
@@ -11,7 +11,11 @@
   describe "POST create" do
     context "sending emails" do
       after {ActionMailer::Base.deliveries.clear}
-      before {ActionMailer::Base.deliveries.clear}
+      let(:charge) {double(:charge, successful?: true)}
+      before do
+        ActionMailer::Base.deliveries.clear
+        StripeWrapper::Charge.stub(:create).and_return(charge)
```

Comment:
I am making this change on all the specs for the users controller right now and one of them causes a failure but passes when using stub... would you be able to explain why that is? Or point me to an explanation? Thanks!
```Failures:
  1) UsersController POST create sending emails does not send out email with invalid inputs
     Failure/Error: expect(StripeWrapper::Charge).to receive(:create).and_return(charge)
       (<StripeWrapper::Charge (class)>).create(any args)
           expected: 1 time with any arguments
           received: 0 times with any arguments
     # ./spec/controllers/users_controller_spec.rb:17:in `block (4 levels) in <top (required)>'```